modules/javafx.graphics/src/main/java/com/sun/glass/ui/monocle/NativePlatform.java

```diff
@@ -129,4 +149,19 @@ public synchronized AcceleratedScreen getAcceleratedScreen(int[] attributes)
         return accScreen;
     }
 
+
+    /**
+     * Log the name of the supplied native cursor class if required.
+     *
+     * @param cursor the native cursor in use, null is permitted
+     * @return the passed in cursor
+     */
+    protected NativeCursor logSelectedCursor(final NativeCursor cursor) {
+        if (debugCursor) {
+            final String name = cursor == null ? null : cursor.getClass().getSimpleName();
+            System.err.println("Using native cursor: " + name);
+        }
```

Comment:
Here you would call `PlatformLogger.fine` unconditionally, for example, instead of the conditional call to `System.err.println`.